src/cogent3/app/evo.py

```diff
@@ -292,10 +294,8 @@ def __call__(self, other, *args, **kwargs):
         return other
 
 
-@define_app
-class model_collection:
-    """Fits a collection of models. Returns a
-    model_collection_result."""
+class _ModelCollectionBase:
```

Comment:
Do we need to give this base class to users of composable decorator?

Reply:
No, it has only those 2 uses.